recipe/meta.yaml

```diff
@@ -10,19 +10,24 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:                # [osx and arm64]
-    - fix_ffi_fcfc.patch  # [osx and arm64]
+  patches:  # [osx]
```

Comment:
The `# [osx]` selector here seems redundant.

Reply:
Thanks. I've just removed the selector